examples/IBFE/explicit/ex4/input2d

```diff
@@ -4,7 +4,7 @@ RHO = 1.0
 L   = 1.0
 
 // grid spacing parameters
-MAX_LEVELS = 1                                      // maximum number of levels in locally refined grid
+MAX_LEVELS = 2                                      // maximum number of levels in locally refined grid
```

Comment:
Should we avoid changing the behavior of the examples?

Reply:
That was an accident. I will revert it.